src/features/pastelID/PastelID.tsx

```diff
@@ -25,19 +25,35 @@ function passphraseStatusColor(validation: TPasswordStrengthResult) {
   return colors[validation.id]
 }
 
+type AddressesWithBalanceProps = {
```

Comment:
Please add prefix `T` to the new defined types

Reply:
@namnm I have updated the prefix. Please check